draft-ietf-tls-tls13.md

```diff
@@ -5023,14 +5030,13 @@ in \[].
         |           |                 | Recv CertificateVerify
         |           +> WAIT_FINISHED <+
         |                  | Recv Finished
-        \                  |
+         \                 | K_recv = application
                            | [Send EndOfEarlyData]
                            | [Send Certificate [+ CertificateVerify]]
```

Comment:
As above, this goes:

* Recv Finished
* [K_send = early data]
* [Send EndOfEarlyData]
* K_send = handshake
* [Send Certificate [+ CertificateVerify]]
* Send Finished
* K_send = application
* K_recv = application (sure, whatever)